src/components/login/ProviderLink.tsx

```diff
@@ -11,15 +11,24 @@ interface Props {
 }
 
 function loginMethod(provider: LoginProvider) {
```

Comment:
I guess this is obsolete now? If I don't misread it, it only accesses `name`, which you can do much easier inline.